joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -421,7 +421,7 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State &)
     auto it = std::find(
       allowed_interface_types_.begin(), allowed_interface_types_.end(), interface);
     if (it == allowed_interface_types_.end()) {
-      RCLCPP_ERROR(logger, "State interface type '" + interface + "' not allowed!");
+      RCLCPP_ERROR(logger, ("State interface type '" + interface + "' not allowed!").c_str());
```

Comment:
```suggestion
      RCLCPP_ERROR(logger, "State interface type '" + interface + "' not allowed!");
```
removing change as #200 addresses this bit